common/src/main/java/com/microsoft/identity/common/internal/platform/AndroidPlatformUtil.java

```diff
@@ -61,6 +66,28 @@
     @Nullable
     private final Activity mActivity;
 
+    /**
+     * List of System Browsers which can be used from broker, currently only Chrome is supported.
```

Comment:
Why only chrome for now?  Also, is this work specific to adding support to default browser to the broker?

Reply:
I just moved this comment wholesale with this method.